### DISCUSSION TITLE
Ẁ

### DISCUSSION THREAD

**Discussion by justanotherfoundry** [koeberlin/Latin-Character-Sets#12] [General]
Do you have any information about how Ẁ and ẁ are used? I found it very difficult to find out anything about its use, if there is any real-world use at all (it practically doesn’t occur in my text corpora). Thanks! Tim

**Comment by koeberlin** [comment]
eki.ee and Hyperglot say its needed for Welsh: http://www.eki.ee/letter/chardata.cgi?search=w+with+grave

**Comment by justanotherfoundry** [comment]
Ah, thanks! I finally found out a bit more about this. Wiktionary has this handy search function: [Welsh terms spelled with Ẁ](https://en.wiktionary.org/wiki/Category:Welsh_terms_spelled_with_%E1%BA%80). Seems like there is only one real word, mẁg (which means mug) and its [soft mutation](https://www.twinkl.de/teaching-wiki/welsh-mutations), fẁg.

Does it mean it is important enough for a “small” character set? It doesn’t take much effort (and storage space), though. Also, we really want Welsh to be shown in Mac’s FontBook, don’t we? AFAIK, that’s based on the Unicode CLDR, which also lists Ẁ as required for Welsh.

How are the Latins S/M/L sets curated? Is every character hand-picked? I really appreciate this project – it seems most designers just throw in all kinds of characters without checking their use and, as far as I understand, this project aims at improving this.

**Reply by koeberlin** [reply]
Thank you, Tim! Good thoughts and questions, as always.
Consider Latin S a modified Latin Extended A where I kept all characters that made sense, even if they’re not widely used, as long as the additional effort is manageable (= you have the base glyph, you have the anchor, you have the accent).
Latin M and Latin L are actual hand-picked selections, and Latin XL – as mentioned – just fills up Adobe Latin 5.
Any feedback always welcome!

**Reply by justanotherfoundry** [reply]
Thanks for the explanation! I will definitely give it a go and then see whether I have additional comments.
